tests/fixtures/test_synthetic_hf_state.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Tuple
```

Comment:
Sorry, I think I misunderstood earlier about what this PR is accomplishing. Looking at it more closely, I do think the original split makes sense -- to put the `synthetic_hf_state` fixture in its own file (inside `tests/fixtures`), and the `test_synthetic_hf_state()` test in a separate file (outside `tests/fixtures`). I don't have a preference on whether `synthetic_hf_state` lives in `tests/fixtures/new_fixtures.py` or its own file.